src/components/Collection.js

```diff
@@ -197,31 +197,36 @@ export default function Collection(props) {
   };
   const refresh = async (_sort, _collectionFilter) => {
     const _api = extjs.connect("https://boundary.ic0.app/", props.identity);
+    var r
```

Comment:
Can these be `let` instead of `var`?

- [ ] prefer `let`

Reply:
absolutely!